DB/projects.json

```diff
@@ -3481,5 +3481,20 @@
         "tech": ["flutter", "hive", "dart"]
     }
   ]
-  }
+  },
+{
+	"github_username": "blindaks",
+	    "Projects": [
+	      {
+	        "github_url": "https://github.com/blindaks/Quiz-App",
+	        "title": "Quiz App",
+	        "description": "A quiz app based on front wnd development questions.",
+	        "tech": [
+	          "html",
+	          "css",
+	          "javascript"
+	        ]
+	      }
```

Comment:
just add these samples to code to the previous one you don't need to add full code every time. check the docs for more information

Reply:
Okay